server/data/index.ts

```diff
@@ -1,13 +1,3 @@
-/* eslint-disable import/first */
-/*
- * Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
- * In particular, applicationinsights automatically collects bunyan logs
- */
-import { initialiseAppInsights, buildAppInsightsClient } from '../utils/azureAppInsights'
-
-initialiseAppInsights()
```

Comment:
This is not needed as instrumentation happens in `server.ts` before anything
(this probably wasn't required before either)